### PR TITLE
Add `inverted` prop to `Slider`

### LINKED ISSUE #216: Suggestion: Reversed Slider
It would be nice to be able to have the slider rendered "reversed". This would be useful for selecting lower boundaries:
reverse

### PR DESCRIPTION
**Pull request by rufman**
If `inverted` is passed to the Slider component, the slider is render
"reversed" with the track going right to left instead of left to right.

fixes #216

### CODE REVIEW

File: examples/slider.js [hunk 1]
```diff
@@ -100,6 +100,22 @@ ReactDOM.render(
       <Slider dots step={20} defaultValue={100} onAfterChange={log} dotStyle={{ borderColor: 'orange' }} activeDotStyle={{ borderColor: 'yellow' }} />
     </div>
     <div style={style}>
+      <p>Inverted Basic Slider</p>
+      <Slider inverted onChange={log} />
```
Comment (benjycui) [inline examples/slider.js]: The style of Slider had been inverted, but the value is not, you can check it locally.

And this means that you unit test need to be updated, too.

Comment (GeorgeSokolovsky) [reply examples/slider.js]: I've check locally and seems like all work as expected.

**Comment by GeorgeSokolovsky** [comment]
Any updates?

**Comment by rufman** [comment]
I'll try to look at this sometime this week and add some more tests

File: examples/slider.js (hunk 1 above)
Comment (paranoidjk) [reply examples/slider.js]: @GeorgeSokolovsky Noop. `Inverted` means when you move slide handler form right to left, the value goes from 0 to 100.

**Comment by breneser** [comment]
Thank you for this work folks. Will this be merged in soon ?

**Comment by joelwatson** [comment]
I could really use this functionality as well! Any updates on status? Anything I can do to assist? Thanks!

File: examples/slider.js (hunk 1 above)
Comment (yesmeck) [reply examples/slider.js]: @rufman Could you check this example? Seems the value is not inverted.

**Comment by iamsoorena** [comment]
this feature is must-have.

**Comment by iamsoorena** [comment]
I've found the best temporary [alternative](https://github.com/sghall/react-compound-slider).

**Comment by arickho** [comment]
Let's merge this in.

**Comment by krzysztofradomski** [comment]
Any chances of having this soon?

**Comment by luzushka** [comment]
Was this merged? I try adding inverted and the slider doesn't change it's behavior / style :(

**Comment by rufman** [comment]
Now invalid: https://github.com/react-component/slider/pull/472

**Pull request closed without merging**